casper/src/main/scala/coop/rchain/casper/engine/NodeLaunch.scala

```diff
@@ -181,6 +181,7 @@ object NodeLaunch {
       conf.genesisBlockData.quarantineLength,
       conf.genesisBlockData.numberOfActiveValidators,
       conf.genesisBlockData.posMultiSigPublicKeys,
+      conf.genesisBlockData.posVaultPubKey,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.